From 5c2e2ddcd0d4574d1f8be11a92e37846d6d85a8a Mon Sep 17 00:00:00 2001
From: "Bernhard M. Wiedemann" <bwiedemann@suse.de>
Date: Fri, 9 Jun 2023 08:10:56 +0200
Subject: [PATCH] MAINT: Sort input file list

so that mumble_flags.qrc builds in a reproducible way
in spite of non-deterministic filesystem readdir order

See https://reproducible-builds.org/ for why this is good.
---
 scripts/generate_flag_qrc.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/generate_flag_qrc.py b/scripts/generate_flag_qrc.py
index c1c5447674..d13f2a9af8 100755
--- a/scripts/generate_flag_qrc.py
+++ b/scripts/generate_flag_qrc.py
@@ -39,7 +39,7 @@ def main():
     # Get a list of all flag SVGs
     flags = []
 
-    for fn in os.listdir(args.flag_dir):
+    for fn in sorted(os.listdir(args.flag_dir)):
         if not fn.lower().endswith('svg'):
             continue
         
